src/main/kotlin/org/elm/lang/core/psi/elements/ElmBinOpExpr.kt

```diff
@@ -0,0 +1,16 @@
+package org.elm.lang.core.psi.elements
+
+import com.intellij.lang.ASTNode
+import org.elm.lang.core.psi.ElmAtomTag
+import org.elm.lang.core.psi.ElmExpressionTag
+import org.elm.lang.core.psi.ElmPsiElementImpl
+import org.elm.lang.core.psi.directChildren
+
+
+/**
+ * One or more binary operator expressions
+ */
+class ElmBinOpExpr(node: ASTNode) : ElmPsiElementImpl(node), ElmExpressionTag {
+    val operators: Sequence<ElmOperator> get() = directChildren.filterIsInstance<ElmOperator>()
```

Comment:
You'll need to replace these two properties with a single sequence of both operators and atoms:
```kotlin
/** The operands and operators in this expression */
val parts: Sequence<ElmExpressionPartTag> get() = directChildren.filterIsInstance<ElmExpressionPartTag>()
```

Iterating over them separately isn't usually useful.